test cases/common/203 function attributes/meson.build

```diff
@@ -65,6 +64,7 @@ endif
 if host_machine.system() != 'darwin'
   attributes += 'alias'
   attributes += 'visibility'
+  attributes += 'alloc_size'
```

Comment:
This seems like an unrelated change, it should be in its own MR.

Reply:
without that, i can't run the tests